index.html

```diff
@@ -4,13 +4,91 @@
 <head>
   <meta charset="utf-8">
   <title>Great Idea - Responsive I</title>
-
+  
   <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
   <link rel="stylesheet" href="css/index.css">
 
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
 </head>
 
 <body>
-<!-- Use past code you created here for Great Idea Web Page or past solution for Great Idea Web Page HTML here! -->
+  <!-- Use past code you created here for Great Idea Web Page or past solution for Great Idea Web Page HTML here! -->
+
+  <div class="container">
+
+    <header>
+      <div class="head-top">
+        <nav>
+          <a href="#">Services</a>
+          <a href="#">Product</a>
+          <a href="#">Vision</a>
+          <a href="#">Features</a>
+          <a href="#">About</a>
+          <a href="#">Contact</a>
+        </nav>
+        <img class="logo" src="https://tk-assets.lambdaschool.com/c7e8c14d-d7ad-4d21-9c18-4a68c77e5ad5_logo.png" alt="Great Idea! Company logo.">
+      </div>
+    </header>
```

Comment:
So, this is mostly preference but, it is never necessary to have a tag just wrapping one element or one tag, in this case, the header wrapping a single div. a single div or header to wrap the nav and image would do the trick.